src/clientsettingsdlg.cpp

```diff
@@ -722,11 +724,11 @@ void CClientSettingsDlg::UpdateJitterBufferFrame()
     // update slider value and text
     const int iCurNumNetBuf = pClient->GetSockBufNumFrames();
     sldNetBuf->setValue ( iCurNumNetBuf );
-    lblNetBuf->setText ( tr ( "Size: " ) + QString().setNum ( iCurNumNetBuf ) );
+    lblNetBuf->setText ( tr ( "Size: " ) + QString::number ( iCurNumNetBuf ) );
```

Comment:
Maybe this could be done with arg() too. 
```suggestion
    lblNetBuf->setText ( QString ( tr ( "Size: %1" ) ) .arg ( iCurNumNetBuf ) );
```

Similar below. 

But I‘m not a Qt expert.

Reply:
Just `tr("").args()` will be fine :)

```suggestion
    lblNetBuf->setText ( tr ( "Size: " ).arg ( iCurNumNetBuf );
```